cms/jinja2/component_overrides/header/_macro.njk

```diff
@@ -20,7 +20,8 @@
 
     {% if params.language and params.language.languages %}
         {% set currentLanguage = params.language.languages | selectattr("current") | first %}
-        {% set currentLanguageIsoCode = currentLanguage.isoCode %}
+        {% set otherLanguages = params.language.languages | rejectattr("current") %}
```

Comment:
no longer needed?

Reply:
Addressed in [26ec8a2](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/26ec8a2434af2b896adaa35b8122c2cea9cbfd1a)